services/pypi/pypi-typing.service.js

```diff
@@ -0,0 +1,40 @@
+import PypiBase, { pypiGeneralParams } from './pypi-base.js'
+
+export default class PypiTyping extends PypiBase {
+  static category = 'platform-support'
+
+  static route = this.buildRoute('pypi/typing')
+
+  static openApi = {
+    '/pypi/typing/{packageName}': {
+      get: {
+        summary: 'PyPI - Typing',
+        description:
+          'Whether the package provides type information (inline annotations or stub files)',
+        parameters: pypiGeneralParams,
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'typing' }
+
+  static render({ isTyped }) {
+    if (isTyped) {
+      return {
+        message: 'typed',
+        color: 'brightgreen',
+      }
+    } else {
+      return {
+        message: 'no',
```

Comment:
I feel like the words here should be opposites: "yes" and "no", or "typed" and "untyped". "typed" and "no" doesn't seem like the right options.

`types | yes` and `types | no` would be consistent with what we've done on the PyPI Wheel badge
https://github.com/badges/shields/blob/00c73c872d9baa5b3e22c658b8d8b73b8b311f7d/services/pypi/pypi-wheel.service.js#L20-L32 which is conceptually similar